rpc_jobs/rpc_artifact_build.yml

```diff
@@ -74,7 +76,7 @@
     # TEMPLATE
     name: 'RPC-Artifact-Build_{series}-{image}-{context}-{ztrigger}'
     project-type: workflow
-    concurrent: false
+    concurrent: true
```

Comment:
This will mean every job is concurrent. Did you mean `concurrent: {concurrent}` ? You'd also need to add `concurrent: true` to template defaults.